### ISSUE TITLE
Email addresses truncated when first typing them in To: field

### ISSUE THREAD

**Issue by jdaviescoates** [the-djmaze/snappymail#513]
**Describe the bug**
When I furst type a contact's email address into the To field I cannot see the full email addresses. The second time it works fine, but never the first time.

![can't_see_email_addresses](https://user-images.githubusercontent.com/2113743/188610664-9a58166d-b2bd-49a2-b6ec-d277d00c8fae.jpg)

**To Reproduce**
1. Hit New message button
2. Start typing the email address of one of my contacts
3. Only see the beginning of the emails addresses, making it impossible to know which is the one I want.
4. Select any of the random email addresses I can only half-see, then delete it.
5. Then start typing the same email address and see what I wanted to see in the first place, the whole email address

**Expected behavior**
I expect to see the full email address the first time, not only after deleting it and then typing it again.

**Please complete the following information:**
- Browser: Firefox 104.0.1
- IMAP daemon: dovecot
- SnappyMail Version: 2.17.4

**Comment by the-djmaze** [comment]
Is this related/duplicate of #368 ?

**Comment by jdaviescoates** [comment]
This seems to have been resolved in 2.18.0 - thanks!

**Comment by jdaviescoates** [comment]
Oh, spoke too soon, sorry. Refreshed and tested again and was same as before again. I guess the next release...

**Comment by the-djmaze** [comment]
Solved with v2.18.2?

**Comment by jdaviescoates** [comment]
Just updated. Seems to be resolved, yes. I'll close for now and reopen if I spot it again. Thanks!
